echemdb/make_pages.py

```diff
@@ -78,7 +78,7 @@ def get_page_links( property_propertyvals_dict ):
             filename = get_echemdb_id_file(echemdb_id)
             ln = f'{echemdb_id}'
 
-    return f'[{ln}](../../../{filename})'
+    return f'[{ln}](../../{filename}/)'
```

Comment:
I would not be surprised if this breaks some other links. But anyway, let's try :)

Reply:
Yeah, could be just wanted to see if it helps something. At least I got back a bit into the code...